src/ui/server/controllers/insurance/business/company-details/validation/companies-house-response/rules/apiError.ts

```diff
@@ -0,0 +1,22 @@
+import { ERROR_MESSAGES } from '../../../../../../../content-strings';
+import { FIELD_IDS } from '../../../../../../../constants';
+import generateValidationErrors from '../../../../../../../helpers/validation';
+import { CompanyHouseResponse } from '../../../../../../../../types';
+
+const {
+  EXPORTER_BUSINESS: { COMPANY_HOUSE },
+} = FIELD_IDS.INSURANCE;
+
+const apiError = (responseBody: CompanyHouseResponse, errors: object) => {
+  let updatedErrors = errors;
+
+  // if flag is set, then error with companies house API or if responseBody is totally empty
+  if (!Object.keys(responseBody).length || responseBody.apiError) {
```

Comment:
Can we check if `responseBody` actually exists

Reply:
done